src/ExtendedXmlSerializer/ExtendedXmlSerializer.csproj

```diff
@@ -56,7 +56,7 @@
     <PackageReference Include="NReco.LambdaParser" Version="1.0.10" />
     <PackageReference Include="Sprache" Version="2.2.0" />
     <PackageReference Include="System.Interactive" Version="4.0.0" />
-    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.6.0" />
     <PackageReference Include="System.Collections.Immutable" Version="1.6.0" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```

Comment:
why not use[ version 4.6](https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/4.6.0)?

Reply:
Yeah good point, @MagicAndre1981... I somehow got an out-of-date version.  Our `Microsoft.SourceLink.GitHub` was also out of date, so went ahead and did that as well.